tests/migration_test.go

```diff
@@ -1143,6 +1144,8 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				cfg := getCurrentKv()
 				cfg.MigrationConfiguration.BandwidthPerMigration = resource.NewMilliQuantity(1, resource.BinarySI)
 				tests.UpdateKubeVirtConfigValueAndWait(cfg)
+
+				migrationBandwidth = resource.MustParse("1Ki")
```

Comment:
@iholder-redhat the failures are "good". 40Mi would be a better pick.

Reply:
Can you explain?

Reply:
I have a feeling 1Ki is not going to be what you would expect(but we can fix this in another PR). The second thing is that 1Gi/1Ki seems odd to ever succeeds fast enough. Therefore I propose 40Mi.

Reply:
> I have a feeling 1Ki is not going to be what you would expect
What do you mean by that?

> The second thing is that 1Gi/1Ki seems odd to ever succeeds fast enough. Therefore I propose 40Mi.
We want to limit the bandwidth to be very low to delay the migration giving it the opportunity to transform into post-copy mode. 40Mi it a lot (roughly 40K) faster then 1Ki and is pretty fast. How would it help us delaying the migration?

Am I missing something here?

Reply:
> > I have a feeling 1Ki is not going to be what you would expect
> 
> What do you mean by that?

This will probably end up being 1Mi as we are converting it to Mi.

> 
> > The second thing is that 1Gi/1Ki seems odd to ever succeeds fast enough. Therefore I propose 40Mi.
> 
> We want to limit the bandwidth to be very low to delay the migration giving it the opportunity to transform into post-copy mode. 40Mi it a lot (roughly 40K) faster then 1Ki and is pretty fast. How would it help us delaying the migration?
> 
> Am I missing something here?

I agree that speed is relative but 40Mi isn't much to 1Gi. Right now you are slowing the migration too much. Have a look at the migration status.

Reply:
Sure, trying to run with 40Mi now. :crossed_fingers: 

> Have a look at the migration status.
Where are you looking at ?

Reply:
:+1 on the analysis of @xpivarc

Reply:
Well, it passed! let's hope it keeps passing!

@xpivarc @rmohr I still don't fully understand why `40Mi` is better than `1Mi` in our case. If we're delaying the migration too much it still needs to transform to post-copy after a while, doesn't it? Even if it causes migration to fail I would expect the test failing on that, not on the transformation to post-copy.

Also @xpivarc I'm interested in what you meant by `Have a look at the migration status`, can you elaborate?

Thanks very much for helping out with this!

Reply:
> @xpivarc @rmohr I still don't fully understand why `40Mi` is better than `1Mi` in our case. If we're delaying the migration too much it still needs to transform to post-copy after a while, doesn't it? Even if it causes migration to fail I would expect the test failing on that, not on the transformation to post-copy.

The point is that even in post-copy it has to transfer the whole content at least once. If you throttle it enough the migration will still take a very long time and will time out eventually.


> Well, it passed! let's hope it keeps passing!

Can you please run the test e.g. 30 times in a loop locally and report back?

Reply:
> The point is that even in post-copy it has to transfer the whole content at least once. If you throttle it enough the migration will still take a very long time and will time out eventually.

Got it, thank you.
If that's the case maybe we should also fix the test so it would check if migration is progressing and not stuck.

> Can you please run the test e.g. 30 times in a loop locally and report back?

Sure, will do

Reply:
@rmohr btw I don't think the bandwidth limit is applied to post-copy. I think there is another argument for it.

>Also @xpivarc I'm interested in what you meant by Have a look at the migration status, can you elaborate?

**[Here](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/6641/pull-kubevirt-e2e-k8s-1.21-sig-compute/1455199075744878592)**  you can go through artifacts and see that [here](https://storage.googleapis.com/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/6641/pull-kubevirt-e2e-k8s-1.21-sig-compute/1455199075744878592/artifacts/k8s-reporter/1_vmis.log) the migration is in post-copy.